easybuild/easyconfigs/p/PyTorch-Ignite/PyTorch-Ignite-0.4.11_fix_error_on_importing_Events.patch

```diff
@@ -0,0 +1,19 @@
+Fix bug error on importing Events in python 3.11
+as in https://github.com/pytorch/ignite/pull/2907
```

Comment:
Since this patch comes directly from upstream we should use the original file. Please use the patch from  https://github.com/pytorch/ignite/pull/2907.patch